lib/ex_ark/utilities.ex

```diff
@@ -20,6 +20,6 @@ defmodule ExArk.Utilities do
   def reverse_binary(bin, len) when is_binary(bin) do
     res = bin |> :binary.decode_unsigned(:little) |> :binary.encode_unsigned(:big)
     pad = len - byte_size(res)
-    <<res::binary, 0::pad*8>>
+    <<0::pad*8, res::binary>>
```

Comment:
This only produces an issue when the UUID component starts with `00`, e.g., `00112233-4455-6677-8899-aabbccddeeff`, or `00112233-4455-6600-8899-aabbccddeeff`.